public/index.html

```diff
@@ -4,7 +4,9 @@
 <head>
   <meta charset="utf-8" />
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel="stylesheet" href="index.css" />
```

Comment:
@Alexandria Instead of adding a stylesheet this way you should follow [create react apps guidelines](https://facebook.github.io/create-react-app/docs/adding-a-stylesheet).